### DISCUSSION TITLE
Running microdnf update in container?

### DISCUSSION THREAD

**Discussion by stianst** [keycloak/keycloak#10172] [Archive]
I question if we need to/should run microdnf update in the container. Should we not be able to expect the ubi-minimal base-image to be updated regularly, and most importantly to fix CVEs?

https://github.com/keycloak/keycloak/blob/cfddcad3c574e1d732b756d8b67aef828b7bb842/quarkus/container/Dockerfile#L24

**Comment by hmlnarik** [comment]
We do, since sometimes an up-to-date image is not available in repositories while RPM with the fixes is available in the repositories used for the updates.

**Reply by stianst** [reply]
Need some more evidence to back that statement. Are you saying that ubi-minimal is not rebuilt when there are CVEs? If that's the case it's not really a good base image to use

**Reply by hmlnarik** [reply]
No. I'm saying that there is a delay between the CVE fix is available in RPM and publicly incorporated in ubi-minimal

**Comment by daurrutia** [comment]
How often are the tagged Quay Keycloak images rebuilt?
